app/experimenter/nimbus-ui/src/components/PageHome/index.tsx

```diff
@@ -13,6 +13,7 @@ import {
   useSearchParamsState,
 } from "../../hooks";
 import { ReactComponent as DownloadIcon } from "../../images/download.svg";
+import { ReactComponent as CreateNewLogo } from "../../images/logo.svg";
```

Comment:
I think we can just call this Logo. WDYT?

```suggestion
import { ReactComponent as Logo } from "../../images/logo.svg";
```